### PR TITLE
Cleaned concepts and forms

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/concepts/concept_void.csv [hunk 1]
```diff
@@ -2,8 +2,6 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:f
 ,TRUE,,Yes,,,,,,Misc,N/A,,,,,,,,,,,
 ,TRUE,,No,,,,,,Misc,N/A,,,,,,,,,,,
 ,TRUE,,Month(s),,,,,,Misc,N/A,,,,,,,,,,,
-,TRUE,,Year(s),,,,,,Misc,N/A,,,,,,,,,,,
 ,TRUE,,Hour(s),,,,,,Misc,N/A,,,,,,,,,,,
-,TRUE,,Day(s),,,,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/concept_void.csv:7]: Let's investigate why that concept is not created by the liquibase.xml file in the first hand.

File: configuration/concepts/commons_void.csv [hunk 2]
```diff
@@ -1,33 +1,24 @@
 Uuid,Void/Retire,Fully specified name:en,_version:1,_order:50
```
Comment (rbuisson) [inline configuration/concepts/commons_void.csv:1]: Let's merge that file and **concept_void.csv** to a **voids.csv** file

File: configuration/concepts/drugs_concepts_void.csv [hunk 3]
```diff
@@ -1,12 +1,4 @@
 Uuid,Void/Retire,Fully specified name:en,_version:1,_order:50
 ,TRUE,Tablet,,
 ,TRUE,Tablet(s),,
-,TRUE,Solution,,
```
Comment (rbuisson) [inline configuration/concepts/drugs_concepts_void.csv:4]: Same thing as above, let's make sure they are not created by Bahmni itself (**liquibase.xml**)

File: configuration/concepts/lab_tests.csv [hunk 4]
```diff
@@ -1,29 +1,134 @@
...
+3f330744-e90c-4a32-9048-07dd255782eb,,CIEL:1643,Rapid malaria,Rapid malaria,,,,,LabTest,Coded,,"Positive; Negative; Indeterminate ",,,,,,,,,
+571e8f4f-4d98-40eb-bd55-c88b8a8ab76d,,CIEL:45,Urine pregnancy test,Urine pregnancy test,,,,,LabTest,Coded,,"Positive; Negative; Indeterminate ",,,,,,,,,
+b2a75309-4eb5-414d-9f29-8ecb4ac7dd32,,CIEL:161456,Bacterial vaginosis,Bacterial vaginosis,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+74ab27c4-c745-45ff-b7e6-e8dbaf933458,,CIEL:163648,Trichomonas,Trichomonas,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+14e99cf4-d294-4905-8837-40e704d8a6ad,,,Vaginal Candidiasis,Vaginal Candidiasis,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+ab6588f6-87ee-45ca-a6c5-dc1337f83469,,CIEL:1036,Gonorrhea,Gonorrhea,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+637b4f48-8717-45e2-a5f8-96869e3bb768,,CIEL:163618,Chlamydia,Chlamydia,,,,,LabTest,Coded,,Positive; Negative; Indeterminate,,,,,,,,,
+4369ee0e-5b3d-4af4-b28b-5cc5cfc34d91,,,Miscellaneous,Miscellaneous,,,,,LabTest,Text,,,,,,,,,,,
+ef5d7595-87f6-4814-95e0-41533a517783,,,Other tests,Other tests,,,,,Sample,N/A,,,Miscellaneous;,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Let's remove those tests that were provided in Haiti. We'll replace them when merging with HSC lab tests.

File: configuration/concepts/lab_tests.csv [hunk 5]
```diff
@@ -1,29 +1,134 @@
...
+18121013-fca2-4bf5-a74c-d4bb7ae4bf4d,,CIEL:690,A+,A+,A+,A+,,,Misc,N/A,,,,,,,,,,,
+58663b05-3dc6-4bff-a6e1-62b4564f8c81,,CIEL:692,A-,A-,A-,A-,,,Misc,N/A,,,,,,,,,,,
+f4fd6b2c-7cea-489d-b623-53bb3842d137,,CIEL:694,B+,B+,B+,B+,,,Misc,N/A,,,,,,,,,,,
+b295f926-3240-4da1-9422-e3ef4c8c3886,,CIEL:696,B-,B-,B-,B-,,,Misc,N/A,,,,,,,,,,,
+e1dc0b09-e7c4-4f0b-9463-58aed51fa06e,,CIEL:699,O+,O+,O+,O+,,,Misc,N/A,,,,,,,,,,,
+4e600262-f421-447d-9e6d-ec78cdf4e041,,CIEL:701,O-,O-,O-,O-,,,Misc,N/A,,,,,,,,,,,
+28380b0e-cbd3-45f7-9421-21113993d7d7,,CIEL:1230,AB+,AB+,AB+,AB+,,,Misc,N/A,,,,,,,,,,,
+064acd44-7bf4-4f19-8133-139234188b03,,CIEL:1231,AB-,AB-,AB-,AB-,,,Misc,N/A,,,,,,,,,,,
+00ce9624-11e3-4015-b695-d7ff7cb3930e,,CIEL:300,Blood Group,Groupe sanguin,Blood Group,Groupe sanguin,,,LabTest,Coded,,A+;A-;B+;B-;O+;O-;AB+;AB-,,,,,,,,,
+6c456eb8-65bf-4d1a-8481-f505d0f19c0a,,,Straw yellow,Jaune paille,Straw yellow,Jaune paille,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Let's apply _Title Case_ here

File: configuration/concepts/lab_tests.csv [hunk 6]
```diff
@@ -1,29 +1,134 @@
...
+09db95e4-185c-4a22-9464-10c77dff27be,,CIEL:159994,Urine,Urine,Urine,Urine,,,Sample,N/A,,,Urine pregnancy test;Urine Colour;Urine Appearance;Density;pH;Blood in urine;Nitrites;Glucose;Acetone Ketone;Albumin;Ascorbic acid;Bilirubin;Urobilinogen;Protein;Urine Macroscopy;Leucocytes;White blood cells;Red blood cell;Bacteria;Epithelail cells;Budding yeasts;Simple yeasts;Mycelium;Macrophages;Kidney cells;Leukocyte cylinders;Granular cylinders;Hyaline cylinders;Erythrocyte cylinders;Crystals;Trichomonas vaginalis;Hématies;Urine Microscopy,,,,,,,,
+faabf960-2025-42e1-8eed-f7825cfa6e2d,,CIEL:162614,Swab,Swab,Swab,Swab,,,Sample,N/A,,,Bacterial vaginosis; Trichomonas; Vaginal Candidiasis; Gonorrhea; Chlamydia;Pap smear,,,,,,,,
+7d14260b-2000-4863-9d8b-71dcb3a32f36,,CIEL:161451,Stool,Stool,Stool,Stool,,,Sample,N/A,,,Stool Consistence;Stool Appearance;Stool Colour;Stool Parasites;Stool Microscopy;H. Pylori;Stool Analysis,,,,,,,,
+,TRUE,,Lab Samples,,,,,,,,,,,,,,,,,,
+,TRUE,,All Orderables,,,,,,,,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/lab_tests.csv]: Let's move that to **voids.csv**

File: configuration/concepts/patient_document.csv [hunk 7]
```diff
@@ -1,4 +1,5 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
+,TRUE,,Document,Document,,Misc,Text,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/patient_document.csv]: Same thing, let's move to **voids.csv**

File: configuration/concepts/registration.csv [hunk 8]
```diff
@@ -1,4 +1,4 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high, version:1, order:1100
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
```
Comment (rbuisson) [inline configuration/concepts/registration.csv:1]: :D

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/concepts/commons_void.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/commons_void.csv:1]: Done

File: configuration/concepts/drugs_concepts_void.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/drugs_concepts_void.csv:4]: Done

File: configuration/concepts/lab_tests.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/patient_document.csv (hunk 7 above)
Comment (reagan-meant) [reply configuration/concepts/patient_document.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

File: configuration/concepts/lab_tests.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/lab_tests.csv]: Done

**Pull request merged**